examples/pretrained_model_inference.ipynb

```diff
@@ -0,0 +1,245 @@
+    "- uaa_clientsecret: The clientsecret used for authentication to the UAA server provided in the __uaa__ of the service key json file\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Environment specific configuration\n",
+    "url = \"\"\n",
+    "uaa_url = \"\"\n",
+    "uaa_clientid = \"\"\n",
+    "uaa_clientsecret = \"\"\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# update working directory path\n",
+    "\n",
+    "import os\n",
+    "\n",
+    "os.chdir('../')\n",
+    "\n",
+    "print(os.getcwd())\n",
+    "\n",
+    "import pathlib\n",
+    "pathlib.Path().absolute()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sap_ber_client import ber_api_client\n",
+    "from pprint import pprint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import importlib\n",
+    "importlib.reload(ber_api_client)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "config_file = os.getcwd() + '/examples/config.json'\n",
+    "with open(config_file, 'rb') as config:\n",
+    "    config_json = json.load(config)\n",
+    "    config.close()\n",
+    "    \n",
+    "config_json"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Instaniate object used to communicate with DC REST API\n",
+    "\n",
+    "url = config_json['url']\n",
+    "uaa_clientid = config_json['uaa']['clientid']\n",
+    "uaa_clientsecret = config_json['uaa']['clientsecret']\n",
+    "uaa_url = config_json['uaa']['url']\n",
+    "\n",
+    "my_ber_client = ber_api_client.BER_API_Client(url, uaa_clientid, uaa_clientsecret, uaa_url)\n",
+    "\n",
+    "print(my_ber_client.base_url)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Display access token"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Token can be used to interact with e.g. swagger UI to explore BER API\n",
+    "print(my_ber_client.session.headers)\n",
+    "print(\"\\nYou can use this token to Authorize here and explore the API via Swagger UI: \\n{}api/v1/\".format(url))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inference\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# post inference job\n",
+    "\n",
+    "#predefined model details as below\n",
+    "model_name = \"sap_invoice_header\"\n",
```

Comment:
Please use model "sap_email_business_entity" as sap_invoice_header is not predicting any values from the example json file in case of batch inference. This was a request from the PM team colleagues that we have data such that some results are predicted.